proto/marshal/rules/message.py

```diff
@@ -34,7 +34,7 @@ def to_proto(self, value):
             try:
                 # Try the fast path first.
                 return self._descriptor(**value)
-            except TypeError as ex:
+            except (TypeError, ValueError) as ex:
```

Comment:
Please also add a link to a bug to add more context on why we need to handle ValueError